From: Jacob Keller <jacob.e.keller@intel.com>
Date: Tue, 14 Jan 2020 12:09:18 -0800
Subject: devlink: fix typos in qed documentation
Patch-mainline: v5.6-rc1
Git-commit: 1ccf6c13d9c7282466f46189e9a1a2ed7f19816a
References: jsc#SLE-12878

Review of the recently added documentation file for the qed driver
noticed a couple of typos. Fix them now.

Noticed-by: Michal Kalderon <mkalderon@marvell.com>
Fixes: 0f261c3ca09e ("devlink: add a driver-specific file for the qed driver")
Signed-off-by: Jacob Keller <jacob.e.keller@intel.com>
Signed-off-by: David S. Miller <davem@davemloft.net>
Acked-by: Thomas Bogendoerfer <tbogendoerfer@suse.de>
---
 Documentation/networking/devlink/qed.rst |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

--- a/Documentation/networking/devlink/qed.rst
+++ b/Documentation/networking/devlink/qed.rst
@@ -22,5 +22,5 @@ The ``qed`` driver implements the follow
    * - ``iwarp_cmt``
      - Boolean
      - runtime
-     - Enable iWARP functionality for 100g devices. Notee that this impacts
-       L2 performance, and is therefor not enabled by default.
+     - Enable iWARP functionality for 100g devices. Note that this impacts
+       L2 performance, and is therefore not enabled by default.
